Resources/views/MediaAdmin/list.html.twig

```diff
@@ -35,9 +35,11 @@ file that was distributed with this source code.
 {% endmacro %}
 
 {% block list_table %}
+    {% if datagrid.values['category'] is defined and datagrid.values['category']['value'] is defined %}
```

Comment:
Oh wow twigfiddle exists. [And I don't think the check is required](http://twigfiddle.com/xqfvu6)